koala/ast/graph.py

```diff
@@ -89,16 +88,27 @@ def prune_graph(self, inputs):
                                 done.add((pred,n))
                     else:
                         if pred.address() not in new_cellmap:
-                            const_node = Cell(pred.address(), pred.sheet, value=pred.value, formula=None, is_named_range=pred.is_named_range, always_eval=pred.always_eval)
-                            pystr,ast = cell2code(self.named_ranges, const_node, pred.sheet)
-                            const_node.python_expression = pystr
-                            const_node.compile()     
-                        else:
-                            const_node = new_cellmap[pred.address()]
+                            const_node = Cell(pred.address(), pred.sheet, value=pred.value, formula=None, is_named_range=False, always_eval=False)
```

Comment:
`Cell` signature considers by default `formula`, `is_named_range` and `always_eval` respectively as `None`, `False`, and `False`.
You could then simplify the code.